src/S3.ts

```diff
@@ -62,21 +63,11 @@ class S3 {
         }
     }
 
-    public async downloadIfPossible(upstreamUrl: string, requestUrl: string): Promise<any> {
+    public async downloadIfPossible(upstreamUrl: string): Promise<any> {
```

Comment:
This should be:
```
public async downloadBlob(key: string): Promise<any>
```

Reply:
I don't really understand why `deleteBlob` has an other API than `downloadBlob`. I believe it should be:
```
public async deleteBlob(key: string): Promise<any>
```